decidim-consultations/app/views/decidim/consultations/questions/_vote_button.html.erb

```diff
@@ -10,28 +10,24 @@
         </div>
       </div>
     <% elsif question.consultation.finished? && signed_in? && question.voted_by?(current_user) %>
-      <%= action_authorized_button_to :unvote,
-                                      decidim_consultations.question_question_votes_path(question),
-                                      method: :delete,
-                                      remote: true,
-                                      data: { disable: true },
-                                      class: "card__button button expanded success disabled",
-                                      id: "unvote_button" do %>
+      <%= button_to decidim_consultations.question_question_votes_path(question),
```

Comment:
Why was this changed? The interface should be the same and the only change would be that the modal is ajaxly loaded, right?

Reply:
Yes, you are right that is not strictly related to this change. Is the same fix than [this one](https://github.com/decidim/decidim/pull/3513/files) but in the consultations module. Both consultations and initiatives are not components, but participatory spaces, so they don't have authorizations. I understand that these helpers are only used to show the login modal if needed. 

For these cases, I defined the `logged_button_to` and `logged_link_to` and used them in this file and in the `initiatives` module templates. 

But in this very specific case, this is not needed, because it is only rendered if the user is signed in. I guess that is the same reason why @mrcasals [has done this](https://github.com/decidim/decidim/pull/3513/files#diff-6cc6b6a8fe819a100d862ce07d40cabfR6).

Reply:
Aha, understood! :+1: